vllm/v1/worker/gpu_input_batch.py

```diff
@@ -45,6 +45,22 @@ def num_tokens(self) -> int:
         return len(self.prompt_token_ids) + len(self.output_token_ids)
 
 
+def _swap_dict_values(obj: dict[int, Any], key1: int, key2: int) -> None:
```

Comment:
Nice :-)

Maybe move this to [utils.py](https://github.com/vllm-project/vllm/blob/main/vllm/utils.py) though?

Reply:
Moved with unit test